# Fix line spacing in logs to be easier to read (#750755)
diff -up pm-utils-1.4.1/pm/pm-functions.in.orig pm-utils-1.4.1/pm/pm-functions.in
--- pm-utils-1.4.1/pm/pm-functions.in.orig	2011-11-02 11:43:24.000000000 +0100
+++ pm-utils-1.4.1/pm/pm-functions.in	2011-11-02 11:43:49.987116603 +0100
@@ -199,9 +199,9 @@ _run_hook() {
 	log "Running hook $*:"
 	hook_ok "$1" && "$@"
 	hook_status=$?
-	log ""
 	log -n "$*: "
 	hook_exit_status $hook_status && LAST_HOOK="${1##*/}" || inhibit
+	log ""
 }
 
 if profiling; then
